lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -16,6 +16,122 @@ module.exports = [
       }, {
         id: 'wordpress',
       }],
+      TagsBlockingFirstPaint: [
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dobetterweb/dbw_tester.css?delay=100',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/unknown404.css?delay=200',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=2200',
+          },
+
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_partial_a.html?delay=200',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
+            mediaChanges: [
+              {
+                href: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
+                media: 'not-matching',
+                matches: false,
+              },
+              {
+                href: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
+                media: 'screen',
+                matches: true,
+              },
+            ],
+          },
+        },
+        {
+          tag: {
+            tagName: 'SCRIPT',
+            url: 'http://localhost:10200/dobetterweb/dbw_tester.js',
+          },
+        },
+        {
+          tag: {
+            tagName: 'SCRIPT',
+            url: 'http://localhost:10200/dobetterweb/fcp-delayer.js?delay=5000',
+          },
+        },
+        undefined, // marks end of array
+      ],
+      // TagsBlockingFirstPaint: {
```

Comment:
which one?

I very much dislike both, but I dislike the first one less, even though it would technically pass something like `[Object,Object,Object,undefined,Object,Object]` ...

Reply:
option c: we could start asserting the array length in our smokes :) 

I'm fine with the first one though, definitely not a huge fan of option 2e

Reply:
i like `c`. I can follow up after this PR.

Reply:
cool beans want to remove the dead weight ones for now then? :)

Reply:
done